app/assets/stylesheets/base.css.scss

```diff
@@ -86,7 +86,7 @@ a.classic-link {
     width: 96%;
 
     .row-label,
-    .inner-section .row-label {
+    .row-label {
```

Comment:
In this file, some CSS selectors might be duplicated (like here). I see at least three occurrences.